charts/ccd-data-store-api/values.template.yaml

```diff
@@ -35,7 +35,8 @@ java:
 
     DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_ps,probate_backend,divorce_ccd_submission,sscs,sscs_bulkscan,cmc,cmc_claim_store,jui_webapp,pui_webapp,bulk_scan_orchestrator,fpl_case_service,iac,finrem_ccd_data_migrator
 
-    DEFINITION_CACHE_TTL_SEC: 259200
+    DEFINITION_CACHE_MAX_IDLE_SEC: 259200
+    DEFINITION_CACHE_LATEST_VERSION_TTL_SEC: 30
```

Comment:
this file is used by AKS, so I think at the moment is Preview and in the end will be applied to AAT as well. So I think this should be 1 sec

Reply:
Changed to 1 for AAT. Thank you!